files/besu/smart_contracts/scripts/keys.js

```diff
@@ -33,5 +33,16 @@ module.exports = {
       url: "http://127.0.0.1:18545",
       accountAddress: "9b790656b9ec0db1936ed84b3bea605873558198"
     }
-  }
+  },
+  accounts: {
+    "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73" : {
+      "privateKey" : "8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63",
+    },
+    "0x627306090abaB3A6e1400e9345bC60c78a8BEf57" : {
+      "privateKey" : "c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3",
+      },
+    "f17f52151EbEF6C7334FAD080c5704D77216b732" : {
```

Comment:
Maybe add "0x" in front of this one as well?

Reply:
Nice catch :)